docker-compose.yml

```diff
@@ -1,6 +1,14 @@
 version: '3.8'
 
 services:
+  database:
+    image: postgres:9.6.21-alpine
+    volumes:
+      - ./database/:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
```

Comment:
Dane do bazy będą na publicznym githubie, więc można je to wrzucić do `.env` file, a plik dodać do `.gitignore`

[https://docs.docker.com/compose/environment-variables/](https://docs.docker.com/compose/environment-variables/)

Reply:
Nice catch, przeniesione